pybinder/core.py

```diff
@@ -1562,6 +1566,22 @@ def is_operator(self):
             return self.spelling in PY_OPERATORS
         return False
 
+    @property
+    def is_getter_method(self):
+        """
+        Check if binder is a public method that returns a reference and the
+        class has an another public method public method named "Set<name>".
+
+        :return: *True* if getter, *False* otherwise.
+        :rtype: bool
+        """
+        if self.is_cxx_method and self.is_public and self.rtype.spelling.endswith("&"):
```

Comment:
is there not a way to check if the return type is by reference in a more sophisticated way? does the property "is_lvalue" apply here?